src/domain/common/repositories/ConstantRepository.ts

```diff
@@ -1,7 +1,9 @@
+import { Code } from "../entities/Base";
 import { Constant } from "../entities/Constant";
 import { SaveOptions } from "../entities/SaveOptions";
 import { Stats } from "../entities/Stats";
 
 export interface ConstantRepository {
+    get(): Promise<Code[]>;
```

Comment:
Normally a repository work with an Entity. In this case the `get` method should return an array of `Constant[]`.